docs/application/resource.md

```diff
@@ -170,6 +192,16 @@ View the component list in the resource detail page:
 
 For more details on dependency graphs, please refer to [Dependency Graph](/application/graph).
 
-![resource-component-graph](/img/v0.5.0/application/resource/resource-component-graph.png)
+![resource-component-graph](/img/v0.6.0/application/resource/resource-component-graph.png)
+
+![resource-component-graph2](/img/v0.6.0/application/resource/resource-component-graph2.png)
+
+## Resource Runs
+
+Every operation such as deployment, rollback, start, and stop of a resource will generate a run record. You can view all run records and their details on the resource details page.
```

Comment:
Every operation, including deployment, rollback, starting, and stopping a resource, generates a run record.

Reply:
updated.